Shared/Samples/Realistic lighting and shadows/README.md

```diff
@@ -0,0 +1,30 @@
+# Realistic lighting and shadows
```

Comment:
Please rename the sample to `Show realistic light and shadows`. There was an effort to standardize the names across platforms, and the old iOS sample name deviates from the common design. Thanks.

- the folder name
- README title
- title in metadata
- swift filename
- view type name
- screenshot name and filename